src/components/Profile/PasswordChangeForm.tsx

```diff
@@ -104,9 +104,12 @@ export const PasswordChangeDialog: FC<PasswordChangeDialogProps> = ({open, close
         </form>
       }
       actions={
-        <Button type="submit" onClick={handleSubmit(onSubmit)}>
-          Potvrdiť
-        </Button>
+        <>
+          <Button type="submit" onClick={handleSubmit(onSubmit)}>
+            Potvrdiť
+          </Button>
+          <Button onClick={close}>Zavrieť</Button>
```

Comment:
bolo by este vhodne pustit `reset()` aby sa vymazali udaje z formulara.

Ak si to chces vyskusat, tak vypln nejake udaje, daj zavriet a otvor si zmenu hesla znova, idealne by mala byt prazdna, no momentalne nie je.